ui/README.md

```diff
@@ -85,12 +85,14 @@ if you've made API changes in `/internal/server` and want to use those on the fr
 
 - MacOS only: `brew install gnu-sed` then follow the instructions to replace the default `sed`
 - Download [the 1.1.2 release of `mockery`](https://github.com/vektra/mockery/releases/tag/v1.1.2) and install in your `/go/bin` directory
-- install `ts-protoc-gen`: `yarn global add ts-protoc-gen` or `npm i -g ts-protoc-gen`
-- install `protoc-gen-grpc-web`: `brew install protoc-gen-grpc-web`
+- Install [`protoc` 3.17.3](https://github.com/protocolbuffers/protobuf/releases/tag/v3.17.3)
```

Comment:
It still feels a little weird that we’re using a different version of protoc for the UI libs, but let’s not rock that particular boat right now. Also: protoc is now at 3.19, so we’re behind the times for both core and UI.